src/libvirtApi/helpers.js

```diff
@@ -89,6 +89,18 @@ export const Enum = {
     VIR_MIGRATE_OFFLINE: 1024,
 };
 
+export const domainEvent = {
+    Defined: 0,
+    Undefined: 1,
+    Started: 2,
+    Suspended: 3,
+    Resumed: 4,
+    Stopped: 5,
+    Shutdown: 6,
+    PMsuspended: 7,
+    Crashed: 8
+};
+
```

Comment:
I was more thinking of incorporating it into the `Enum` variable above as following:
```
VIR_DOMAIN_EVENT_DEFINED: 0,
VIR_DOMAIN_EVENT_UNDEFINED: 1,
VIR_DOMAIN_EVENT_STARTED: 2,
VIR_DOMAIN_EVENT_SUSPENDED: 3,
VIR_DOMAIN_EVENT_RESUMED: 4,
VIR_DOMAIN_EVENT_STOPPED: 5,
VIR_DOMAIN_EVENT_SHUTDOWN: 6,
VIR_DOMAIN_EVENT_PMSUSPENDED: 7,
VIR_DOMAIN_EVENT_CRASHED: 8,
VIR_DOMAIN_EVENT_LAST: 9,
```
source: https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainEventType

Reply:
Ah, I see. will fix. Seemed a little weird to mash everything into the same type... the whole point of enums is to be *different* types?